src/databricks/labs/ucx/assessment/crawlers.py

```diff
@@ -13,6 +14,20 @@
     "spark.databricks.hive.metastore.glueCatalog.enabled",
 ]
 
+AZURE_SP_CLUSTER_CONF = ["fs.azure.account.key",
+                         "fs.azure.account.auth.type",
+                         "fs.azure.account.oauth.provider.type",
+                         "fs.azure.account.oauth2.client.id",
+                         "fs.azure.account.oauth2.client.secret",
+                         "fs.azure.account.oauth2.client.endpoint"]
+
+AZURE_CLUSTER_POLICY = ["spark_conf.fs.azure.account.auth.type",
```

Comment:
These keys are duplicates from the previous array, but with prefix. Generate them dynamically when you need them